### PR TITLE
feat(pier): add pier start with binary and docker

### PR DESCRIPTION
**Pull request by Alexader**
1. download specific binary pier to start pier
2. check if old fabric is running when starting a new fabric

### CODE REVIEW

File: scripts/run_pier.sh [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    mkdir -p "${PIER_ROOT}"/plugins
+
+    # copy appchain crypto-config and modify config.yaml
+    print_blue "===> Copy fabric crypto-config"
+    if [ ! -d "${CURRENT_PATH}"/crypto-config ]; then
+        print_red "crypto-config not found, please start fabric network first"
+        exit 1
+    fi
+    cp -r "${CURRENT_PATH}"/crypto-config "${PIER_ROOT}"/fabric/
+    x_replace "s/10.1.16.48/localhost/g" "${PIER_ROOT}"/fabric/config.yaml
```
Comment (sandyzhou) [inline scripts/run_pier.sh]: the replacement is not needed since it was done in another PR

Comment (Alexader) [reply scripts/run_pier.sh]: Ok, i will fix it.

**Comment by jzhe886** [comment]
LGTM

**Review by jzhe886 — APPROVED** [review]

**Review by sandyzhou — APPROVED** [review]
